solana-programs/staking-bridge/programs/staking-bridge/src/wormhole_utils.rs

```diff
@@ -232,6 +231,7 @@ pub fn execute_wormhole_transfer(
 
     // https://github.com/wormhole-foundation/wormhole/blob/main/solana/modules/token_bridge/program/src/lib.rs#L107
     let instruction_index: u8 = 4;
+    let fee: u64 = 0;
```

Comment:
yep, thanks. this is indeed what we do in prod right now for our other usages of wormhole.

the reason they have this is because they wanted to build something (may exist already) to incentivize others to complete the transfer through the wormhole so you do not have to claim it. in this case, np though. you've added nice scripts!